src/Microsoft.AspNetCore.SystemWebAdapters.FrameworkServices/Microsoft.AspNetCore.SystemWebAdapters.FrameworkServices.csproj

```diff
@@ -28,6 +28,10 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.DataProtection" Version="6.0.8" />
+    <PackageReference Include="Microsoft.AspNetCore.DataProtection.Extensions" Version="6.0.8" />
+    <PackageReference Include="Microsoft.Owin.Security.Cookies" Version="4.2.2" />
```

Comment:
This will force everyone to have the owin dependency even if they don't need it.  Should we have a package that specifically is for the Owin related helpers?

Reply:
At that point, maybe just leaving the code in the sample and adding it to the docs is a better option?